server/controllers/items/items.coffee

```diff
@@ -11,7 +11,6 @@ module.exports =
       'by-ids': require './by_ids'
       'by-users': require './by_users'
       'by-entities': require './by_entities'
-      'last-public': require './last_public'
```

Comment:
we actually still need this endpoint for the `last public` inventory layout
![last_public](https://user-images.githubusercontent.com/1596934/32399862-0efffd22-c0fa-11e7-8101-8df6d3e5dbe1.png)
It might be made obsolete by the incoming [inventory browser](https://github.com/inventaire/inventaire-client/tree/inventoryBrowser) view, but it's too early to tell, so better let it around for now